src/audio/aria/aria.c

```diff
@@ -170,7 +170,7 @@ static void aria_set_stream_params(struct comp_buffer *buffer,
 
 static int aria_prepare(struct processing_module *mod,
 			struct sof_source **sources, int num_of_sources,
-			struct sof_sink __sparse_cache **sinks, int num_of_sinks)
```

Comment:
I would suggest squash this with previous one, since they do same thing.

Reply:
all prev sink/source commits were separate, but at this point you're right, separation of commits does not bring any value here.